etrago/cluster/electrical.py

```diff
@@ -477,6 +477,12 @@ def select_elec_network(etrago, apply_on="grid_model"):
     ----------
     etrago : Etrago
         An instance of the Etrago class
+    apply_on: str
+        gives information about the objective of the output network. If
+        "spatial_clustering" is provided, the value assigned in the args for
```

Comment:
Is it correct that you can choose 'spatial_clustering' here? 
The code later on only allows 'grid_model' or 'market_model'